### ISSUE TITLE
Display of response header name is altered when header name and value are case-insensitive matches

### ISSUE THREAD

**Issue by gdub** [Huachao/vscode-restclient#709]
- REST Client Version: 0.24.3
- VSCode Version: 1.50.1
- OS Version: Ubuntu 20.04

Steps to Reproduce:
1. Return a response with header and value that compare as case-insensitive equals, e.g.:
    ```
    Server: SerVer
    My-Header: MY-HEADER
    ```
2. REST Client mangles the header names in its display of the response:
    ```
    SerVer: SerVer
    MY-HEADER: MY-HEADER
    ```
I would expect the case of the header names to match what came back in the raw response.

**Comment by Huachao** [comment]
@gdub nice catch, I have fixed the bug and will publish in the next release.

**Comment by Huachao** [comment]
@gdub I have published a new version 0.24.4. and you can have a try.
